### DISCUSSION TITLE
When configured with ACR for image signing  validation , connaisseur pods are getting terminated and restarted continuously with error CrashLoopbackoff error

### DISCUSSION THREAD

**Discussion by RamBhatt-gmail** [sse-secure-systems/connaisseur#1683] [Q&A]
I have followed the documentation to add ACR with cosign validator to validate the docker images uploaded , but connaisseur  pods keep getting crashed, due to which image validation. Is not working, none of the images I can deploy. Please do let me know what I am doing wrong.
Here is the configuration 
![IMG_20240709_152148]

(https://github.com/sse-secure-systems/connaisseur/assets/175108791/8a56cb3b-3672-4b8b-b91a-ac59a14be7f6)

On further debug logs from pods, the error seen is - "error loading config error parsing file no authentication file for secret connaisseur " , this is with secret name  even though secrets are created in aks cluster and is available ,   I changed it to use username and password in yaml, then I get error - " error loading config : error parsing file: neither secret name not use keychain defined ".

**Comment by Starkteetje** [comment]
Hi @RamBhatt-gmail 
which version of Connaisseur are you running?

Generally the `no authentication file for secret` occurs when Connaisseur is not able to find the authentication file for the validator. What I find weird is that you get `connaisseur` as the secret's name in the log file, while your config lists `connaisseur-acr-access-2`, which don't align.

With the newest version of Connaisseur, the container shouldn't even start up, giving 
```
Warning  FailedMount  22s (x8 over 86s)  kubelet            MountVolume.SetUp failed for volume "<secret>-volume" : secret "<secret>" not found
```
or similar if the wrong secret name is set.

To help you, it would be nice if you tried the setup with the newest Connaisseur version and could provide an example config which fails in addition to the commands you used to setup the secrets.

Kind regard
